src/talk/renderer/components/UserMenu.vue

```diff
@@ -18,54 +79,88 @@
-						<template #default>
-							{{ visibleUserStatus }}
-						</template>
-					</UiMenuItem>
-					<UiMenuItem tag="button" @click.native="reload">
-						<template #icon>
-							<MdiReload />
-						</template>
-						{{ t('talk_desktop', 'Force reload') }}
-					</UiMenuItem>
-					<UiMenuItem tag="a" :href="$options.packageInfo.bugs" target="_blank">
-						<template #icon>
-							<MdiBug />
-						</template>
-						{{ t('talk_desktop', 'Report a bug') }}
-					</UiMenuItem>
-					<UiMenuItem tag="a" :href="talkWebLink" target="_blank">
-						<template #icon>
-							<MdiWeb />
+					<template v-if="userStatusSubMenuOpen">
+						<UiMenuItem tag="button" @click.native.stop="userStatusSubMenuOpen = false">
+							<template #icon>
+								<MdiChevronLeft :size="20" />
+							</template>
+							{{ t('talk_desktop', 'Back') }}
+						</UiMenuItem>
+						<UiMenuItem v-for="status in ['online', 'away', 'dnd', 'invisible']"
+							:key="status"
+							tag="button"
+							@click.native.stop="handleUserStatusChange(status)">
+							<template #icon>
+								<NcUserStatusIcon :status="status" />
+							</template>
+							{{ userStatusTranslations[status] }}
+							<template v-if="status === userStatus.status" #action-icon>
+								<MdiCheck :size="20" />
+							</template>
+						</UiMenuItem>
+					</template>
+
+					<template v-else>
+						<UiMenuItem tag="a"
+							:href="userProfileLink"
+							target="_blank">
+							<div class="user-menu__name">
+								<strong>{{ user['display-name'] }}</strong>
+								<em>{{ user.id }}</em>
+							</div>
+							<span>
+								{{ t('talk_desktop', 'View profile') }}
+							</span>
+						</UiMenuItem>
+
+						<UiMenuSeparator />
+
+						<UiMenuItem tag="a" :href="serverUrl" target="_blank">
+							<template #icon>
+								<ThemeLogo :size="24" />
+							</template>
+							<span class="user-menu__server">
+								<span>{{ theming.name }}</span>
+								<em>{{ serverUrlShort }}</em>
+							</span>
+						</UiMenuItem>
+
+						<UiMenuSeparator />
+
+						<template v-if="userStatus">
+							<UiMenuItem tag="button" @click.native.stop="userStatusSubMenuOpen = true">
+								<template #icon>
+									<NcUserStatusIcon :status="userStatus.status" />
+								</template>
+								{{ userStatusTranslations[userStatus.status] }}
+								<template #action-icon>
+									<MdiChevronRight :size="20" />
+								</template>
+							</UiMenuItem>
+							<UiMenuItem key="custom-status" tag="button" @click.native="isUserStatusDialogOpen = true">
+								<template #icon>
+									<span v-if="userStatus.icon" style="font-size: 20px">
+										{{ userStatus.icon }}
+									</span>
+									<MdiEmoticonOutline v-else :size="20" />
+								</template>
+								{{ userStatus.message || t('talk_desktop', 'Set custom status') }}
+								<template #action-icon>
+									<MdiPencil v-if="userStatus.message" :size="20" />
+									<MdiChevronRight v-else :size="20" />
```

Comment:
Chevron right seems confusing here as It will be a modal open not a submenu.

Reply:
Removed